### PR TITLE
Rewording and fixing capitalizations

### PR DESCRIPTION
**Pull request by ewolinetz**
@soltysh @bparees PTAL

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ at common flags that can be used with all of the subcommands.
 | Name                       | Description                                             |
 |:-------------------------- |:--------------------------------------------------------|
 | `-h (--help)`              | Display help for the specified command |
-| `--loglevel`               | Set log level of log output (0-3) (see [Log levels](#log-levels))|
-| `-U (--url)`               | URL of docker socket to use (default: `unix:///var/run/docker.sock`) |
+| `--loglevel`               | Set the level of log output (0-3) (see [Log levels](#log-levels))|
```
Comment (bparees) [inline docs/cli.md]: is it only 0-3?  i think it's 0-5.

Comment (bparees) [inline docs/cli.md]: we definitely have some V(5) now.  probably didn't used to.

File: docs/cli.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ at common flags that can be used with all of the subcommands.
...
 
 There are four log levels:
-* `Level 0` - produces output from containers running `assemble` script and all encountered errors
-* `Level 1` - produces basic information about the executed process
-* `Level 2` - produces very detailed information about the executed process
-* `Level 3` - produces very detailed information about the executed process, alongside with listing tar contents
+* Level `0` - produces output from containers running `assemble` script and all encountered errors
+* Level `1` - produces basic information about the executed process
+* Level `2` - produces very detailed information about the executed process
+* Level `3` - produces very detailed information about the executed process, alongside with listing tar contents
```
Comment (bparees) [inline docs/cli.md]: Level 4 produces very very detailed
Level 5 produces very very very detailed

ok not realy.
looks like level 4 doesn't add anything and level 5 adds info about credentials used to pull docker images.

**Comment by bparees** [comment]
@soltysh thanks you

please add the info about log level 4/5 and lgtm.

**Comment by ewolinetz** [comment]
@bparees I also updated this as well: https://github.com/openshift/source-to-image/blob/master/cmd/sti/main.go#L257

**Comment by bparees** [comment]
lgtm, thanks @ewolinetz !
[merge]

**Comment by soltysh** [comment]
re-[merge] apparently we're getting the same error as we did before

**Pull request merged**